docs/Serverless.md

```diff
@@ -0,0 +1,82 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing fastify application.
+
+
+## AWS Lambda
+
+On top of AWS Lambda and Amazon API Gateway. The sample provided allows you to easily build serverless web applications/services and RESTful APIs using fastify.
```

Comment:
I don't understand the point of the first sentence here. It seems incomplete.

Reply:
rearranged the sentence